install.sh

```diff
@@ -107,9 +110,14 @@ sudo rm -rf ~/{.npm,.forever,.node*,.cache,.nvm}
 
 if [[ ! -z $DEB ]]; then
     sudo wget --quiet -O - https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
-    (echo "deb https://deb.nodesource.com/node_12.x $(lsb_release -s -c) main" | sudo tee /etc/apt/sources.list.d/nodesource.list)
+        if [[ "$DEB_ID" == "Debian" ]] || [[ "$DEB_ID" == "Ubuntu" ]]; then
+            (echo "deb https://deb.nodesource.com/node_12.x $(lsb_release -s -c) main" | sudo tee /etc/apt/sources.list.d/nodesource.list)
+        elif [[ "$DEB_ID" != "Debian" ]] || [[ "$DEB_ID" != "Ubuntu" ]]; then
```

Comment:
Should be `&&`, not `||` here I think?

Reply:
> Should be `&&`, not `||` on line 115 I think?

Did you test? Should be fine with OR as well 🤔

Reply:
Why do you have `elif` at all instead of `else`?

Reply:
> That condition is always true. Why do you have `elif` at all instead of `else`?

Correct, as long as it's DEB package all conditions are covered, so `else` is just fine. Changed.

Reply:
Generally speaking any condition of that kind (`x != val1 || x != val2`) is true. Usually `&&` is intended instead `||` (that's what @deanpress meant).